ps_checkpayment.php

```diff
@@ -69,7 +69,7 @@ public function __construct()
         if ((!isset($this->checkName) || !isset($this->address) || empty($this->checkName) || empty($this->address))) {
             $this->warning = $this->trans('The "Payee" and "Address" fields must be configured before using this module.', array(), 'Modules.Checkpayment.Admin');
         }
-        if (!count(Currency::checkPaymentCurrencies($this->id))) {
+        if ($this->id === null || !count(Currency::checkPaymentCurrencies($this->id))) {
```

Comment:
I guess this message should be displayed only when the module is installed. It should pass the test only when `id` is different from null

Reply:
```suggestion
        if ($this->id && !count(Currency::checkPaymentCurrencies($this->id))) {
```